qiskit/utils/deprecation.py

```diff
@@ -59,11 +59,18 @@ def _rename_kwargs(func_name, kwargs, kwarg_map):
             if new_arg in kwargs:
                 raise TypeError(f"{func_name} received both {new_arg} and {old_arg} (deprecated).")
 
-            warnings.warn(
-                "{} keyword argument {} is deprecated and "
-                "replaced with {}.".format(func_name, old_arg, new_arg),
-                DeprecationWarning,
-                stacklevel=3,
-            )
-
-            kwargs[new_arg] = kwargs.pop(old_arg)
+            if new_arg is None:
+                warnings.warn(
+                    f"{func_name} keyword argument {old_arg} is deprecated and ignored."
```

Comment:
Should we make this a bit more precise to say this will be ignored in the future? Otherwise it might sound like it is already ignored 🤔 
```suggestion
                    f"{func_name} keyword argument {old_arg} is deprecated and "
                    "will in future be ignored."
```

Reply:
This is my misunderstanding. When I write this, I thought `it is already ignored `... But you are right.

Reply:
Thank you.

Reply:
Is `will in future be removed.` better?

Reply:
Yes I think that would be even better!